src/js/components/FilterInputText.js

```diff
@@ -92,15 +92,15 @@ class FilterInputText extends React.Component {
     });
 
     return (
-      <span className="form-control-group-add-on form-control-group-add-on-append">
+      <span className="form-control-group-add-on">
```

Comment:
Good call, I didn't realize that. I just took a quick glance at what's in `reactjs-components` and I believe that it should be removed from `reactjs-components`. The classname I'm using here comes from CNVS.